src/app/Library/CrudPanel/Traits/Fields.php

```diff
@@ -344,8 +344,6 @@ public function getFields()
      * Check if the create/update form has upload fields.
      * Upload fields are the ones that have "upload" => true defined on them.
      *
-     * @param  string  $form  create/update/both - defaults to 'both'
-     * @param  bool|int  $id  id of the entity - defaults to false
```

Comment:
Note: these arguments have disappeared a long time ago, we just never removed the docblocks.